src/utils/database.ts

```diff
@@ -4407,7 +4570,12 @@ const createCryptoTransaction = async (
     confirmed_at: new Date().toISOString(),
     currency: Currency.USD,
     total_fee: feeInUSD,
-    metadata: {},
+    metadata: {
+      sender_address: account_address.toLowerCase(),
```

Comment:
Why are we storing, account_address twice ? it's already being stored in the intiator address yes ?